tests/cupy_tests/core_tests/test_ndarray.py

```diff
@@ -570,6 +570,11 @@ def test_bytes_tobytes_scalar_array(self, xp, dtype):
         x = xp.array(3, dtype)
         return bytes(x)
 
+    @testing.numpy_cupy_equal()
+    def test_format(self, xp):
+        x = xp.array(1.12345)
+        return x.__format__('.2f')
```

Comment:
```suggestion
        return format(x, '.2f')
```

like `bytes(x)` instead of `x.__bytes__()`.   FYI, `numpy.ndarray` does not have `__bytes__` but `bytes(x)` works.